tests/model_fields/models.py

```diff
@@ -373,6 +375,22 @@ class PersonTwoImages(models.Model):
             width_field="headshot_width",
         )
 
+    class NoReadImageModel(models.Model):
+        """
+        Model that defines an ImageField with a storage backend that does not
+        support reading.
+
+        This supports a test to avoid a double-read performance problem."""
+
+        testimage = models.ImageField(
+            upload_to="tests",
+            storage=NoReadFileSystemStorage(),
+            width_field="width",
+            height_field="height",
+        )
+        width = models.IntegerField()
+        height = models.IntegerField()
```

Comment:
Optional - we could also make a Person-y thing if we want
```suggestion
    class PersonNoReadImage(models.Model):
        """
        Model that defines an ImageField with a storage backend that does not
        support reading.

        This supports a test to avoid a double-read performance problem."""

        mugshot = models.ImageField(
            upload_to="tests",
            storage=NoReadFileSystemStorage(),
            width_field="width",
            height_field="height",
        )
        mugshot_width = models.IntegerField()
        mugshot_height = models.IntegerField()
```